src/main/kotlin/com/dwolla/api/CustomersApi.kt

```diff
@@ -291,6 +291,8 @@ class CustomersApi(private val client: DwollaClient) {
     @Throws(DwollaApiException::class, DwollaAuthException::class)
     fun upgradeToVerifiedPersonal(
         id: String,
+        firstName: String,
+        lastName: String,
```

Comment:
I like this addition, though I'm a little hesitant to place the parameters at the beginning of the list. Intuitively it makes sense and _should not_ pose a problem if the user is using Kotlin; however, since Kotlin is interoperable with Java, if someone is using the SDK with Java, this would break the function call since function arguments are ordered, not named.